execution-order/execution-order.css

```diff
@@ -0,0 +1,58 @@
+html, body {
+  margin: 0;
+  padding: 0;
+}
+#main {
+  display: grid;
+  grid-template-columns: min-content minmax(0, 1fr);
+  height: 100vh;
+}
+#main.ready {
+  animation: slidein .25s ease-in-out;
+}
+.closer {
+  max-height: 100vh;
+}
+.closer::after {
+  content: "\bb";
```

Comment:
The close button is also a unicode character.

Reply:
This is misleading - the `>>` doesn't **collapse** the panel. The panel is **closed**, so you should use the `x` that we use in all of our dialogs. Actually, remove all these custom controls and simply reuse `messageBoxProxy`.

Reply:
Using messageBoxProxy should also solve the problem of background transparency: currently when we drag the style, the empty gap in its initial position shows the underlying text of the main page.